build/buildRuby.sh

```diff
@@ -287,6 +281,8 @@ echo "========================= Performing Building FluentD"
 cd ${FLUENTD_DIR}
 elevate bundle install --local
 elevate bundle exec rake build
+# currently 7 tests are failing
```

Comment:
is this related?

Reply:
Not related, I wanted to enable fluentd unittests, but there are 7 tests already failing, it doesn't have to do with this change.